### DISCUSSION TITLE
Web follower not working

### DISCUSSION THREAD

**Discussion by sammyahmedtech** [NightscoutFoundation/xDrip#2813] [Q&A]
Web follower for Libre 3 just stopped working a couple of days ago. I tried to update to the latest nightly and I also tried different follower accounts. I live in Sweden.

Anyone else having problems or know what is wrong? 
![Uploading Screenshot_2023-04-30-17-14-44-39_2dd2c1bcb8f659cd696bcba2072f751b.jpg…]()

**Comment by Navid200** [comment]
I have explained in the following welcome thread how to post a screenshot:
https://github.com/NightscoutFoundation/xDrip/discussions/1900

Can you install the official link-up follower app.
Does it work?

**Comment by Navid200** [comment]
https://navid200.github.io/xDrip/docs/Libre3_Follow.html

**Comment by PhilTU** [comment]
I also have a non functioning web follower. It is on the same phone as a working instance of LibreLinkUp using the same credentials. It doesn't receive any data or record anything in the events log. I have changed the script name, generated an error and returned it to the correct value. xDrip+ update from the latest stable to early July did not help.

Any extra logging I can turn on ?

**Reply by Navid200** [reply]
Does the official follower work?

**Reply by PhilTU** [reply]
"It is on the same phone as a working instance of LibreLinkUp using the same credentials"

Yes.

**Reply by Navid200** [reply]
Did you add a space after the configuration script?

**Reply by PhilTU** [reply]
I have done so now.

Previously as above I put in another character, now I left it with a trailing space

**Reply by Navid200** [reply]
Now, remove the trailing space.

**Reply by PhilTU** [reply]
Removed, No change. Changed to different user / password credentials too. Official follower app works. xDrip+ is not.

Adding a space didn't generate anything visible in Errors/events log. Adding another character did generate a template lookup error.
